old_tests/RunAll.sql

```diff
@@ -83,12 +83,12 @@ exec ut_coverage.coverage_start_develop();
 @@lib/RunTest.sql ut_reporters/ut_sonar_test_reporter.AcceptsFileMapping.sql
 @@lib/RunTest.sql ut_reporters/ut_sonar_test_reporter.ProducesExpectedOutputs.sql
 @@lib/RunTest.sql ut_reporters/ut_teamcity_reporter.ProducesExpectedOutputs.sql
-@@lib/RunTest.sql ut_reporters/ut_xunit_reporter.ProducesExpectedOutputs.sql
+@@lib/RunTest.sql ut_reporters/ut_junit_reporter.ProducesExpectedOutputs.sql
```

Comment:
Will you be able to migrate the `old_tests` that you modify to the new tests format (using utPLSQL)?

Reply:
Sounds good